spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -2889,6 +2855,31 @@ object QueryPlanSerde extends Logging with CometExprShim {
     }
   }
 
+  def supportedShuffleDataType(dt: DataType): Boolean = dt match {
+    case _: ByteType | _: ShortType | _: IntegerType | _: LongType | _: FloatType |
+        _: DoubleType | _: StringType | _: BinaryType | _: TimestampType | _: TimestampNTZType |
```

Comment:
This code was moved and is now new. I added `TimestampNTZType`.

Reply:
Do we have a test with TimestampNTZType for shuffle?